docs/release_notes.rst

```diff
@@ -16,6 +16,13 @@ New Features
 * Added D404 - First word of the docstring should not be "This". It is turned
   off by default (#183).
 
+* Added the ability to ignore specific function and method doctstrings with
+  comments:
+
+    1. "# noqa" or "# pydocstyle: noqa" skips all checks.
```

Comment:
@Eric89GXL This and `# pydocstyle: D102,D203` are what I'm talking about. You should just use `# noqa` below. Even so, having `# pydocstyle: noqa` in a file will skip the whole file? That's what people are going to expect from having used `# flake8: noqa`. You're going to confuse people with this.